cmd/network/bridge/bridge.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"flag"
+	"os"
+	"strings"
+
+	"github.com/golang/glog"
+	"kubevirt.io/kubernetes-device-plugins/pkg/dpm"
+	"kubevirt.io/kubernetes-device-plugins/pkg/network/bridge"
+)
+
+func main() {
+	flag.Parse()
+
+	bridgesListRaw, bridgesListDefined := os.LookupEnv("BRIDGES")
+	if !bridgesListDefined {
+		glog.Exit("BRIDGES environment variable must be set in format BRIDGE[,BRIDGE[...]]")
+	}
+
+	bridgesList := strings.Split(bridgesListRaw, ",")
+
+	manager := dpm.NewDevicePluginManager(bridge.BridgeLister{Bridges: bridgesList})
```

Comment:
This looks like reversed-reversed responsibility: it is the lister that should be able to figure out the default bridges (but the env var check is in the right place).

Reply:
Done